report_pre_thesis/src/sections/moc.tex

```diff
@@ -44,10 +44,49 @@ \subsubsection{Multivariate Oxide Composition Derivation}\label{sec:moc_derivati
+\begin{equation}
+    h_t = \text{diag}\left[ t(t^T t)^{-1} t^T \right]
+\end{equation}
+
+Given that leverage quantifies the distance of each observation from the model's center, it can be interpreted as the square Mahalanobis distance — a measure of the distance of a point from the center of mass of points in multivariate space. 
+An important property of the Mahalanobis distances is the fact that, if the original data follows a multivariate normal distribution, the distances form a chi-squared distribution. Using this property, outliers can be removed using the chi-square test \cite{brereton_chi_2015}.
+The Mahalanobis distance is defined as follows:
+
+\begin{equation}
+    D_M(p)^2 = (p - \mu)^T \cdot \Sigma^{-1} (p - \mu)
+\end{equation}
+
+Where $p$ is the point in question, $\mu$ is the mean of the distribution and $\Sigma^{-1}$ is the inverse of the covariance matrix of the distribution, known as the precision matrix. 
+Variables exhibiting a high leverage score are indicative of predictors that deviate more significantly from the others, thereby have a greater influence on the model.
+Hotelling's \( T^2 \) statistic extends the concept of the squared Mahalanobis distance by incorporating sample size, enabling hypothesis testing. 
+This statistic assesses if a sample's mean vector significantly differs from a hypothesized mean. Hotelling's \( T^2 \) is defined as follows:
+
+\begin{equation}
+    T^2 = n \cdot (\overline{X} - \mu_0)^T \cdot S^{-1}(\overline{X} - \mu_0)
+\end{equation}
+
+where $n$ is the sample size, $\overline{X}$ is the sample mean vector, $\mu_0$ is the hypothesized population mean vector and $S$ is the covariance matrix.
+
+The relationship between Hotelling's \( T^2 \) and leverages can then be expressed as:
+\begin{equation}
+    \text{Hotelling's } T^2 = (N - 1) (h_i - \frac{1}{N})
+\end{equation}
+
+Where $N$ is the sample size. The equation suggests that Hotelling's $T^2$ is proportional to the leverage of an observation, adjusted by the mean ($\frac{1}{N}$) and scaled by $N - 1$, where $N - 1$ represents the degrees of freedom adjustment.
+The measure of model fit, denoted as $Q$, is calculated by squaring the differences between the actual spectrum $x$ and the spectrum predicted by the model. This prediction uses the model's scores $t$ and loadings ($P$) to calculate the residuals $e$:
+
+\begin{equation}
+    e = x - t \cdot P^T
+\end{equation}
+
+The fit of model for the $i$th observation is then computed by taking the residual vector $e_i$ and multiplying it by the transpose of itself $e^T$, which gives you the sum of squared differences: 
+
+\begin{equation}
+    Q_i = e_{i}e_{i}^T
+\end{equation} \cite{marini_chemometrics_2013} \citet{andersonImprovedAccuracyQuantitative2017}
```

Comment:
I'm not sure about this. This section follows the logical chain - with some additional information - that we see in Anderson et al., which doesn't feel right.

Ideally we should just refer to the paper and say that we're doing _that_. The only difference _should_ be that we're using automated Chi-Square testing to detect outliers, which we've already justified. Didn't we also do that in the other PR, where we actually talk about what we've done?

Since we probably need more space for experiments etc., perhaps this is too much detail, and we should just ref? Thoughts?

Reply:
I agree that it does follow the same logical chain, but I thought their paper presumed a lot of knowledge that we have to display that we understand. Recall the previous meeting with our supervisors where they were confused about how this worked. For that reason I chose to include the extra details and I felt that referring to their paper and adding the extra details felt odd.